src/implementations/document-store/transfer-ownership.ts

```diff
@@ -17,16 +17,15 @@ export const transferDocumentStoreOwnershipToWallet = async ({
 }: DocumentStoreTransferOwnershipCommand): Promise<TransactionReceipt> => {
   const wallet = await getWalletOrSigner({ network, ...rest });
   if (dryRun) {
-    const documentStore = await DocumentStoreFactory.connect(address, wallet);
+    const documentStore = await OwnableFactory.connect(address, wallet);
```

Comment:
Why change `DocumentStoreFactory` to `OwnableFactory` in this PR? Do you mind sharing the rationale behind making this change?

Reply:
Oh… No more using `transferOwnership`. Use `grantRole` and `revokeRole` instead if you wish to change someone’s access roles. 

The `transferOwnership` shouldn’t be in the readme. I will update the readme for document store tomorrow when I have time.🤞